### ISSUE TITLE
Converting InceptionV3 to TfLite missing IdentityN

### ISSUE THREAD

**Issue by ngelotte** [tensorflow/tensorflow#27882]
**System information**
Google Colab
- TensorFlow installed from (source or binary):
Installed from https://storage.googleapis.com/download.tensorflow.org/data/tensorflow_hub-0.4.0.dev0-py2.py3-none-any.whl
- TensorFlow version (or github SHA if from source):

**Provide the text output from tflite_convert**
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: AVERAGE_POOL_2D, CONCATENATION, CONV_2D, FULLY_CONNECTED, MAX_POOL_2D, MEAN, RESHAPE, SOFTMAX. Here is a list of operators for which you will need custom implementations: IdentityN.

Also, please include a link to a GraphDef or the model if possible.
Google Colab Notebook: https://colab.research.google.com/drive/1idF4ZZysBnfcGwfoj0DovQb8C4NVA42w

I updated the Google Colab Tutorial and updated it to use InceptionV3(based on the link included in the notebook and modified the file size to be [299,299].

**Comment by ANSHUMAN87** [comment]
@ngelotte : I tried executing your colab link, but it throws error at "**Export your model**" stage, would you please check and resolve it, so that i can debug the issue. Otherwise you can share the frozen pb file directly if you have. Thanks!

**Comment by ngelotte** [comment]
@ANSHUMAN87 : Sorry I modified that same notebook to try and figure out how to freeze the graph. I copied over the network and took out what I had added - you can view it here. https://colab.research.google.com/drive/17e9E56uQpkdaD7Lz-csAOS2PL8Tzoz_1 
However, It now gets a different error and I don't know what has changed. The new error is:
Invalid argument: Input 1 of node keras_layer_1/StatefulPartitionedCall was passed float from Variable_1:0 incompatible with expected resource.

**Comment by ANSHUMAN87** [comment]
@ngelotte : Now i am able to run the notebook, for the error you faced("Invalid argument: Input 1 of node keras_layer_1/StatefulPartitionedCall was passed float from Variable_1:0 incompatible with expected resource."), is because of custom objects from Hub_Layer.

As tf.lite.TFLiteConverter.from_saved_model() does not support custom objects, i would suggest rather you save model in h5 file and use tf.lite.TFLiteConverter.from_keras_model_file("sample.h5", custom_objects={'KerasLayer':hub.KerasLayer}).

As for your original issue: below i have provided snapshot for working code.
converter = tf.lite.TFLiteConverter.from_keras_model_file("sample.h5", custom_objects={'KerasLayer':hub.KerasLayer})
converter.target_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,
                        tf.lite.OpsSet.SELECT_TF_OPS]

tflite_model = converter.convert()


NOTE: In COLAB Notebook some error is reported while saving to h5 file, so you can run offline. Or you can download the saved_model files and convert offline.

**Comment by jvishnuvardhan** [comment]
@ngelotte Is this issue resolved by following @ANSHUMAN87 suggestion? Thanks!

**Comment by ngelotte** [comment]
I was not able to try it as I did not have a good local environment to run it in. As the goal of this conversion was to be able to use it on a EdgeTPU device I am able to follow the steps here https://github.com/tensorflow/tensorflow/issues/27880 to do the conversion. This area is still in active development it seems but I think it is the better route. Thanks for the help.

**Comment by ANSHUMAN87** [comment]
@jvishnuvardhan : I think this issue can be closed now. Thanks!

**Comment by jvishnuvardhan** [comment]
Automatically closing this out since I understand it to be resolved, but please let me know if I'm mistaken.Thanks!

**Comment by miaout17** [comment]
The original reported issue should be resolved by commit aca24307eba. 
Could you try it again with the next TF nightly build? Thanks!
